ui/src/literature/components/ReferenceList.tsx

```diff
@@ -36,11 +36,13 @@ function ReferenceList({
       <ReferenceItem
         key={reference.getIn(['titles', 0, 'title']) || String(index)}
         reference={reference}
+        referenceId={index}
+        total={total}
```

Comment:
can we already calculate this here? instead of passing down the `total`

Reply:
I pass total so that the query for refreshing references after edit has correct page. Example: if we go to page 123 of references in one record, and then go to another record that has only one page of references and edit, value of query.page in state would still be 123 and result in incorrect request, so I do this:
<img width="455" alt="Screenshot 2023-06-14 at 14 27 50" src="https://github.com/inspirehep/inspirehep/assets/55505399/6ca52a1d-cc0b-4cee-aeae-e219d076b871">